readme.md

```diff
@@ -1 +1 @@
-# Job board
+# Job board
```

Comment:
A guide on how to setup locally, explaining your env and other stuff is required